src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Xamarin.Android.Sdk.Lite.props

```diff
@@ -12,6 +12,11 @@
     <UsingAndroidNETSdk />
   </PropertyGroup>
 
+  <ItemGroup>
+    <!-- Remove Microsoft.Android FrameworkReference in favor of default references below. -->
+    <FrameworkReference Remove="Microsoft.Android" />
```

Comment:
Does this need to check some property, like `$(DisableImplicitFrameworkReferences)`?

Reply:
We check `DisableImplicitFrameworkReferences`: https://github.com/xamarin/xamarin-macios/blob/bf88adadafa71f899ca8310555d43407dada63d5/dotnet/targets/Xamarin.Shared.Sdk.DefaultItems.targets#L26

Reply:
This is just for the Lite SDK which will never support a Xamarin flavored @(FrameworkReference), as such a concept is incompatible with the legacy SDK. I don't think we can/should make this conditional.